### PR TITLE
add rocky linux box configuration

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: vagrant/boxes.d/00-rocky.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+boxes:
+  rocky8:
+    box_name: 'rockylinux/8'
+    libvirt: https://dl.rockylinux.org/pub/rocky/8/images/x86_64/Rocky-8-Vagrant-Libvirt.latest.x86_64.box
```
Comment (evgeni) [inline vagrant/boxes.d/00-rocky.yaml:4]: bleh, at least this one doesn't boot properly

Comment (ekohl) [reply vagrant/boxes.d/00-rocky.yaml:4]: Why do we need to specify the URL? Doesn't https://app.vagrantup.com/rockylinux work (like CentOS)?

Comment (evgeni) [reply vagrant/boxes.d/00-rocky.yaml:4]: It works like centos, yes, it yields a 404.

Comment (evgeni) [reply vagrant/boxes.d/00-rocky.yaml:4]: https://github.com/theforeman/forklift/blob/2e259fe13406fc5dbec516cd63f9324660d86731/vagrant/boxes.d/00-centos.yaml#L34

Comment (ekohl) [reply vagrant/boxes.d/00-rocky.yaml:4]: Is that what they mean with bug-for-bug compatible?

Comment (evgeni) [reply vagrant/boxes.d/00-rocky.yaml:4]: :fire:

**Review by ekohl — APPROVED** [review]

**Pull request merged**